pvlib/solarposition.py

```diff
@@ -105,8 +105,8 @@ def get_solarposition(time, latitude, longitude,
                               pressure, temperature,
                               how='numpy', **kwargs)
     elif method == 'pyephem':
-        ephem_df = pyephem(time, latitude, longitude, pressure, temperature,
-                           **kwargs)
+        ephem_df = pyephem(time, latitude, longitude, altitude, pressure,
```

Comment:
the ``pyephem`` function signature uses keyword arguments instead of positional arguments, so we should do the same here just to be safe and consistent.

Reply:
That's wise. I've changed to the kwargs.